src/python/pants/backend/python/macros/pipenv_requirements_caof_test.py

```diff
@@ -131,15 +132,15 @@ def test_supply_python_requirements_file(rule_runner: RuleRunner) -> None:
                 pipfile_target='//:custom_pipfile_target'
             )
 
-            _python_requirements_file(
+            _target_generator_sources_helper(
```

Comment:
Since this will be a somewhat public API via `peek` and `filter`, the naming matters a bit more. Maybe just `_generator_sources`? "target" is implicit since it's a target. And `helper` is mainly used to disambiguate a method/name from some support code, and there is nothing to collide with here.

Reply:
I wanted to keep the `_helper` around to help people intuit better why some target generators have this (`python_requirements`) and others don't (`python_sources`). Are you okay with `_generator_sources_helper`?